tools/convert_pkl_to_pb.py

```diff
@@ -183,25 +213,48 @@ def convert_op_name(op):
         reset_names(op.output)
         return [op]
 
-    @op_filter(type="Python", inputs=['rpn_cls_probs', 'rpn_bbox_pred', 'im_info'])
-    def convert_gen_proposal(op_in):
-        gen_proposals_op, ext_input = convert_gen_proposals(
-            op_in, blobs,
-            rpn_min_size=float(cfg.TEST.RPN_MIN_SIZE),
-            rpn_post_nms_topN=cfg.TEST.RPN_POST_NMS_TOP_N,
-            rpn_pre_nms_topN=cfg.TEST.RPN_PRE_NMS_TOP_N,
-            rpn_nms_thres=cfg.TEST.RPN_NMS_THRESH,
-        )
-        net.external_input.extend([ext_input])
-        return [gen_proposals_op]
+    @op_filter()
```

Comment:
@op_filter(type='Python')?

Reply:
@newstzpz L218 is here to replace it if I am not mistaking

Reply:
I don't really have much preference between a direct check below and a `type='Python'` check in the decorator. Any preference across the team?